azure-kusto-data/azure/kusto/data/__init__.py

```diff
@@ -1,4 +1,148 @@
-from .kusto_client import KustoClient, KustoResultIter, KustoResponse
-from .kusto_exceptions import KustoError, KustoClientError, KustoServiceError
+from enum import Enum
```

Comment:
Please read https://www.python.org/dev/peps/pep-0008/ for how to format Python code (e.g. you should have newlines separating the different classifications of imports).